CMakeLists.txt

```diff
@@ -112,11 +112,40 @@ if(USE_STDERR_LOGGER)
   add_definitions(-DUSE_STDERR_LOGGER=1)
 endif()
 
-option(NON_HERMETIC_TESTS "Whether to build and run tests that depend on an internet connection" OFF)
+option(BUILD_MISC_TESTS "Build additional tests and utilities" OFF)
+option(BUILD_FUN_UTILS "Build addtional just for fun utilities" OFF)
```

Comment:
You should probably enable this in some CI configuration(s) to make sure it doesn't break in the CMake build at some point in the future.

Reply:
I checked, and `BUILD_MISC_TESTS` from which I derived this, aren't built either, should I enable both in cmake-linux?

Reply:
Enabled it in CircleCI ASAN build.